libnavui-util/src/main/java/com/mapbox/navigation/ui/utils/internal/SvgUtil.kt

```diff
@@ -64,6 +64,25 @@ object SvgUtil {
         }
     }
 
+    @Throws
+    fun renderAsBitmapWithWidth(
```

Comment:
This fully duplicates the below function. Could we parametrize `SVGExternalFileResolver` instead (make it nullable)?

Reply:
Fixed.